tests/Console/ConfigurationResolverTest.php

```diff
@@ -1170,9 +1170,15 @@ public function provideDeprecatedFixerConfiguredCases(): array
     public function provideGetDirectoryCases(): array
     {
         return [
-            [null, '/my/path/my/file', 'path/my/file'],
+            [null, '/my/path/my/file', 'my/file'],
+            [null, './my/file', 'my/file'],
+            [null, 'my/file', 'my/file'],
             ['/my/path2/dir/.php-cs-fixer.cache', '/my/path2/dir/dir2/file', 'dir2/file'],
+            ['/my/path2/dir/.php-cs-fixer.cache', './dir2/file', 'dir2/file'],
+            ['/my/path2/dir/.php-cs-fixer.cache', 'dir2/file', 'dir2/file'],
             ['dir/.php-cs-fixer.cache', '/my/path/dir/dir3/file', 'dir3/file'],
+            ['dir/.php-cs-fixer.cache', './dir3/file', 'dir3/file'],
+            ['dir/.php-cs-fixer.cache', 'dir3/file', 'dir3/file'],
```

Comment:
@keradus 

Following your suggestion I added some more test cases. If the path starts with a dot I think that should be removed from the relative path, hence why with `./my/file` I am expecting `my/file`.